diff --git a/node_modules/escpos/index.js b/node_modules/escpos/index.js
index c8570d8..ba490ff 100644
--- a/node_modules/escpos/index.js
+++ b/node_modules/escpos/index.js
@@ -323,8 +323,11 @@ Printer.prototype.tableCustom = function (data, options = {}) {
   } else {
     return this
   }
-}
+};
 
+Printer.prototype.formater = function(data){
+  console.log("formater " + data);
+}
 
 /**
  * [function Print encoded alpha-numeric text without End Of Line]
